coral/.eslintrc.cjs

```diff
@@ -74,6 +74,7 @@ module.exports = {
         "react",
         "@typescript-eslint",
         "no-relative-import-paths",
+        "lodash"
```

Comment:
Maybe a slight git mishap, I see the changes from the previously merged lodash linter rule :o

Reply:
uh, weird. what did I dooooo? 
Will check and update the PR!